Source/santa-driver/SantaDecisionManager.cc

```diff
@@ -14,6 +14,13 @@
 
 #include "SantaDecisionManager.h"
 
+// This is a made-up KAUTH_FILEOP constant which represents a KAUTH_VNODE_WRITE_DATA event
+// that gets passed to SantaDecisionManager's FileOpCallback method.  It is defined as 8
+// because that was the first unused integer from sys/kauth.h.  The reason that we don't
+// simply use the KAUTH_VNODE_WRITE_DATA constant as is is because it overlaps with the other
+// KAUTH_FILEOP  constants.
+#define KAUTH_FILEOP_WRITE 8
```

Comment:
If this is still needed we should give the actions used by KAUTH_SCOPE_FILEOP room to grow. I don't really like the idea of adding our own action, but if we need to make it more than 8 incase fileop adds a new action in the future.

Maybe KAUTH_FILEOP_DELETE + 32 or something

Reply:
Done.